test/common_device_type.py

```diff
@@ -229,6 +278,60 @@ def only_fn(slf, device, *args, **kwargs):
         return only_fn
 
 
+# Lists of Torch types
+DOCUMENTED_TENSOR_TYPES = [torch.float64, torch.float32, torch.float16,
+                           torch.int64, torch.int32, torch.int16, torch.int8,
+                           torch.uint8, torch.bool]
+
+# SCALAR_TYPES = DOCUMENTED_TENSOR_TYPES - half and bool
+# Note: derived from AT_FORALL_SCALAR_TYPES
+SCALAR_TYPES = [torch.float64, torch.float32,
+                torch.int64, torch.int32, torch.int16, torch.int8,
+                torch.uint8]
+
+QUANTIZED_TYPES = [torch.quint8, torch.qint8, torch.qint32]
```

Comment:
I don't think we need these yet -- these are just "stub" types anyway, in that they don't fully specify the value of the tensor.

Reply:
Sounds good.